pkg/apis/samples/v1/types.go

```diff
@@ -328,10 +328,12 @@ func (s *Config) ClearNameInReason(reason, name string) string {
 }
 
 const (
-	noInstallDetailed = "Samples installation in error at %s: %s"
-	installed         = "Samples installation successful at %s"
-	moving            = "Samples processing to %s"
-	removing          = "Deleting samples at %s"
+	noInstallDetailed  = "Samples installation in error at %s: %s"
+	installed          = "Samples installation successful at %s"
+	moving             = "Samples processing to %s"
+	removing           = "Deleting samples at %s"
+	doneImportsFailed  = "Samples installed at %s, with image import failures, see samples operator config object"
+	failedImageImports = "FailedImageImports"
```

Comment:
"Progressing=false, reason=FailedImageImports" still strikes me as a slightly weird thing to report to a user, but if @smarterclayton says that's the preferred to to bubble up the information, i guess this accomplishes that.

Reply:
yeah @bparees my recollection from out discussion was that the reason field on progressing is what was highlighted in the console

Though I have not been able to find that exactly, so maybe I'm off here

Reply:
bump @bparees @smarterclayton 

please confirm if this form of reporting was in fact one of the to-do's for failed imagestream import notification that came out of our disconnected discussions a couple of weeks ago.

Reply:
i think the immediate to do that came out of the meeting was to report this via a new condition on the clusteroperator resource (ie the AttentionRequired condition), but I see no reason not to also have this condition on the samples operator resource, and then use it to drive the update of the clusteroperator resource once we agree on the AttentionRequired condition.

tldr this seems fine.